docs/en/end-user/components/more.md

```diff
@@ -4,6 +4,48 @@ title:  Want More?
 
 Components in KubeVela are designed to be brought by users.
 
+## 1. Get from capability center
+
+KubeVela allows you to explore capabilities maintained by other developer.
+There are two command in kubectl vela plugin: `comp` and `trait`.
+
+In case you haven't install kubectl vela plugin: see [this](../../kubectl-plugin).
+
+### 1. list
+
+For example, let's try to list all availible components in registry:
+
+```shell
+$ kubectl vela comp --discover
+Showing components from default registry:https://github.com/oam-dev/catalog/tree/master/registry
+NAME              	CENTER	DEFINITION                 	
```

Comment:
What does `CENTER` mean?

Reply:
Also let's use a consistent idiom between `capability`, `definition`, `comp/traits`.

Reply:
`CENTER` is so called by vela CLI. I forgot to change it to `REGISTRY` in docs.